3-async-await/solution.js

```diff
@@ -20,16 +20,70 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-function solution() {
+const products = require('./products');
+const prices = require('./prices');
+
+function getId() {
+    return Number(String(Date.now()).slice(-2));
```

Comment:
This does work, but is really uncommon to use constructors for primitive types. You should have done something like
```javascript
parseInt(Date.now().toString().slice(-2))
```

Reply:
Could also use an arrow function for simplicity

```javascript
const calculateRandomId = () => parseInt(Date.now().toString().slice(-2))
```